macros/src/main/scala/gestalt/macros/DefMacros.scala

```diff
@@ -20,6 +20,26 @@ object plusObject {
         toolbox.Lit(null)
     }
   }
+
+  inline def varargs(tped: Int*): Int = meta {
+    val q"$items: $_" = tped
+    items match {
+      case toolbox.SeqLiteral(items: Seq[toolbox.Tree]) =>
+        items.reduceLeft((a, b) => q"$a + $b")
+      case _ =>
+        q"$items.reduce((a:Int,b:Int)=> a + b)"
+    }
+  }
```

Comment:
Handling two cases is fine, I'm just thinking `tpd` could take the type `List[Tree]`, that simplify the usage a lot IMO.